### PR TITLE
RPD unwrench upgrade.

### PR DESCRIPTION
**Pull request by Dennok**
## About The Pull Request

Adds reverse wrench mode upgrade to the RPD.
Attention, due to budget cuts, the mode is hard linked to the destroy mode control button.

## Why It's Good For The Game

OP atmostech is good

## Changelog
:cl:
add: Adds reverse wrench mode upgrade to the RPD. Attention, due to budget cuts, the mode is hard linked to the destroy mode control button.
/:cl:

### CODE REVIEW

**Comment by Ghilker** [comment]
nice

File: code/game/objects/items/RPD.dm [hunk 1]
```diff
@@ -212,6 +212,7 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 	var/static/datum/pipe_info/first_disposal
 	var/static/datum/pipe_info/first_transit
 	var/mode = BUILD_MODE | DESTROY_MODE | WRENCH_MODE
+	var/upgrade = FALSE
```
Comment (bobbahbrown) [inline code/game/objects/items/RPD.dm:215]: This comment (or lack thereof) does not meet our [documentation standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine), please update this.

Comment (bobbahbrown) [reply code/game/objects/items/RPD.dm:215]: Additionally setting the initial value to a FALSE literal is not necessary

**Comment by bobbahbrown** [comment]
Does this upgrade effectively cause a RPD to be unable to destroy pipes after this has been installed?

**Comment by bobbahbrown** [comment]
Your PR is failing an appveyor build. I would recommend testing your code before webediting and PRing it.

**Comment by Jared-Fogle** [comment]
Please create a proper branch and test your code before submitting it instead of web editing.

**Comment by Barhandar** [comment]
>Does this upgrade effectively cause a RPD to be unable to destroy pipes after this has been installed?

It should be able to do both since it destroys loose pipes only. If it doesn't then code bad.

**Pull request closed without merging**